### PR TITLE
Update LocalSettings.php

### PR DESCRIPTION
**Pull request by paladox**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -2096,13 +2096,16 @@
 	),
 	'wgManageWikiPermissionsBlacklistRights' => array(
 		'default' => array(
+			// arrays
```
Comment (JohnFLewis) [inline LocalSettings.php]: why is it necessary to add a comment? there's no comment on the other probably hundred of arrays in this file

File: LocalSettings.php [hunk 2]
```diff
@@ -2096,49 +2096,55 @@
...
+				'hideuser',
+				'interwiki',
+				'managewiki-restricted',
+				'managewiki-editdefault',
+				'mwoauthmanageconsumer',
+				'mwoauthmanagemygrants',
+				'mwoauthsuppress',
+				'mwoauthviewprivate',
+				'mwoauthviewsuppressed',
+				'read',
```
Comment (MacFan4000) [inline LocalSettings.php]: Remove read from any

File: LocalSettings.php (hunk 1 above)
Comment (paladox) [reply LocalSettings.php]: I've removed it now.

File: LocalSettings.php [hunk 3]
```diff
@@ -2096,49 +2096,55 @@
...
+				'renameuser',
+				'requestwiki',
+				'siteadmin',
+				'skipcaptcha',
```
Comment (MacFan4000) [inline LocalSettings.php]: This too

File: LocalSettings.php [hunk 4]
```diff
@@ -2096,49 +2096,55 @@
...
+				'suppressionlog',
+				'suppressrevision',
+				'torunblocked',
```
Comment (MacFan4000) [inline LocalSettings.php]: Same here

**Pull request merged**